tests/test_eval.py

```diff
@@ -159,6 +159,8 @@ def test_show_results(evaluator_mobilenet, test_catdog_dataset_path):
     average_df = evaluator_mobilenet.show_results(mode='average')
     assert average_df['model'][0] == 'catdog-mobilenet.hdf5'
     assert average_df['accuracy'][0] == 0.75
+    assert average_df['sensitivity'][0] == 0.75
+    assert average_df['weighted_precision'][0] == 0.833
```

Comment:
(Being really picky here) shall we list `precision` before `weighted_precision`? 
```suggestion
    assert average_df['precision'][0] == 0.833
```

Reply:
I changed it to accuracy, weighted precision, sensitivity, precision. To reflect first 2 weighted, second 2 without